src/us_reverse_geo/Result.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A candidate is a possible match for an address that was submitted.<br>
+ *     A lookup can have multiple candidates if the address was ambiguous.
+ *
+ * @see "https://smartystreets.com/docs/cloud/us-reverse-geo-api#result"
+ */
+class Result {
+	constructor(responseData) {
+		this.coordinate = {};
+		if (respoonseData.coordinate !== undefined) {
+			this.coordinate.latitude = responseData.coordinate.latitude;
+			this.coordinate.longitude = responseData.coordinate.longitude;
+			this.coordinate.accuracy = responseData.coordinate.accuracy;
+			switch (responseData.coordinate.license) {
+				case 1:
+					this.coordinate.license = "SmartyStreets Proprietary";
+				default:
+					this.coordinate.license = "SmartyStreets";
+			}
+		}
+
+		this.distance = responseData.distance;
+
+		this.address = {};
+		if (responseData.address !== undefined) {
```

Comment:
Same idea here, I think it should be safe to assume a truthy or falsey value here
```suggestion
		if (responseData.address) {
```